ingestion/src/metadata/profiler/orm/functions/unique_count.py

```diff
@@ -53,5 +54,18 @@ def _unique_count_query_mssql(col, session, sample):
     )
 
 
+def _unique_count_query_oracle(col, session, sample):
+    count_fn = col
+    if isinstance(col.type, HexByteString):
+        count_fn = CountFn(col)
+    return (
+        session.query(func.count(count_fn))
+        .select_from(sample)
+        .group_by(count_fn)
+        .having(func.count(count_fn) == 1)
+    )
```

Comment:
Shouldn't this be handled in the `CountFn` itself?

Reply:
func.count(col) gives -> 'count(propagated_msgid)'

whereas when i pass in the Countfn with custom logic for HexByteString i.e. blob as we have mapped it internally, then we get the count as count(DBMS_LOB.GETLENGTH(propagated_msgid))

we have handled it in CountFn, unique count though doesn't take CountFn, rather it uses func.count